.github/workflows/ci.yml

```diff
@@ -115,7 +115,7 @@ jobs:
           container: ubuntu:20.04
         - name: windows_cl-2022_debug_analysis
           os: windows-2022
-          env: {ALPAKA_CI_CXX: cl.exe,  ALPAKA_CI_CL_VER: 2022,                                   CMAKE_BUILD_TYPE: Debug,   ALPAKA_BOOST_VERSION: 1.78.0, ALPAKA_CI_CMAKE_VER: 3.25.3,                     ALPAKA_CI_ANALYSIS: ON, alpaka_DEBUG: 2}
+          env: {ALPAKA_CI_CXX: cl.exe,  ALPAKA_CI_CL_VER: 2022,                                   CMAKE_BUILD_TYPE: Debug,   ALPAKA_BOOST_VERSION: 1.78.0, ALPAKA_CI_CMAKE_VER: 3,                          ALPAKA_CI_ANALYSIS: ON, alpaka_DEBUG: 2}
```

Comment:
Would "3" work for any 3.x installed or it means directly 3.0 ?

Reply:
I assume the package manager we use to install cmake does not supports minor versions and will use something modern.
Lets wait for a response from @SimeonEhrig

Reply:
`3` means only the major version is specified. At the moment CMake 3.31 is preinstalled. If MS is updating the image we get a another version of CMake 3 (probably something newer than 3.31).
I cannot uninstall the preinstalled version. Therefore I cannot decide what we use and I thought CMake 4 will be not released soon the the version `3` shows that we only get CMake 3.